### ISSUE TITLE
Failed to vagrant up on headless system.

### ISSUE THREAD

**Issue by phuonglm** [teracyhq/dev#233]
Try to provision teracy-dev on my headless Ubuntu 14.04.5 LTS x64 system and got this error when `vagrant up`. Vagrant version is 1.9.1 and Virtualbox 5.1.14.  
```
phuonglm@test02:~/Projects/teracy/dev$ vagrant destroy -f && vagrant up
==> default: Using hostname "teracy.dev" as node name for Chef...
==> default: Destroying VM and associated drives...
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Importing base box 'bento/ubuntu-16.04'...
==> default: Matching MAC address for NAT networking...
==> default: Checking if box 'bento/ubuntu-16.04' is up to date...
==> default: Setting the name of the VM: dev_default_1485252746716_33849
==> default: Using hostname "teracy.dev" as node name for Chef...
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
    default: Adapter 2: bridged
==> default: Forwarding ports...
    default: 22 (guest) => 2222 (host) (adapter 1)
==> default: Running 'pre-boot' VM customizations...
==> default: Booting VM...
There was an error while executing `VBoxManage`, a CLI used by Vagrant
for controlling VirtualBox. The command and stderr is shown below.

Command: ["startvm", "46cc798d-446c-430a-b7e2-6ef9d26f7fff", "--type", "headless"]

Stderr: VBoxManage: error: The virtual machine 'dev_default_1485252746716_33849' has terminated unexpectedly during startup with exit code 1 (0x1)
VBoxManage: error: Details: code NS_ERROR_FAILURE (0x80004005), component MachineWrap, interface IMachine
```

**Comment by hoatle** [comment]
```
phuonglm [2:03 PM] 
lỗi vbox anh fix được rồi nhé

[2:03]  
apt-get remove virtualbox* không clean hết mấy module driver

[2:03]  
dpkg -i không check old version

[2:03]  
phải apt-get purge virtualbox* rồi cài lại mới hết
```

**Comment by hoatle** [comment]
so basically, need to clean up virtualbox before installing a new version, I'll mark this: resolved as normal behavior, refer this to #207 to mention about this problem and how to get it fixed.
